MediaRecorder.html

```diff
@@ -222,61 +223,83 @@
             <ol>
               <li>Let <var>timeslice</var> be the method's first argument, if
               provided, or <code>undefined</code>.</li>
-              <li>If <code>state</code> is not <code>inactive</code>, throw an
-              <code>InvalidStateError</code> DOMException and abort these
-              steps.</li>
+              <li>If <a>state</a> is not <a href="#dom-recordingstate-inactive">
+              inactive</a>, throw an <code>InvalidStateError</code> DOMException
+              and abort these steps.</li>
+
               <li>If the stream's isolation properties disallow access from
               this MediaRecorder, throw a <code>SecurityError</code>
               DOMException and abort these steps.</li>
-              <li>Set <code>state</code> to "recording", and run the following
-              steps in parallel:
-                <ol>
 
+              <li>Set <a>state</a> to <a href="#dom-recordingstate-recording">
+              recording</a>, and run the following steps in parallel:
+                <ol>
                   <li>Once media becomes available from one or more of the
-                  stream's tracks, start gathering the data into a Blob (see
-                  [[!FILE-API]]), and queue a task, using the DOM manipulation
-                  task source, to fire an event named <code>start</code>.</li>
+                  stream's tracks, let <var>blob</var> be the Blob
+                  ([[!FILE-API]]) of collected data, start gathering the data
+                  into <var>blob</var> and queue a task, using the DOM
+                  manipulation task source, to <a
+                  href="https://dom.spec.whatwg.org/#concept-event-fire">fire
+                  </a> an event named <code>start</code>.</li>
 
                   <li>If at any point the stream's isolation properties change
                   so that MediaRecorder is no longer allowed access to it, the
                   UA MUST immediately stop gathering data, discard any data that
                   it has gathered, and queue a task, using the DOM manipulation
-                  task source, that <a>fires an error event</a> with the name
-                  <code>SecurityError</code>, immediately followed by a <code>
-                  stop</code> event. </li>
+                  task source, that runs the following steps:
+                  <ol>
+                    <li>Set <a>state</a> to
+                    <a href="#dom-recordingstate-inactive">inactive</a>.</li>
+                    <li><a href="#dfn-firing-an-error-event-named-e">Fire an
+                    error event</a> with the name <code>SecurityError</code>
+                    </li>
+                    <li><a href="https://dom.spec.whatwg.org/#concept-event-fire">
+                    Fire</a> an event named <code>stop</code>.</li>
+                  </ol>
+                  </li>
 
                   <li>If the UA at any point is unable to continue gathering
```

Comment:
Oddly enough there are no existing cases with which to be consistent, so I guess you get to pick the style for the whole file, but now that this is no longer a single paragraph, I'd probably put the first bit in `<p>` on a new line.